pyproject.toml

```diff
@@ -30,7 +30,8 @@ torch = {version = "^2.2.1", source="torch"}
 torchaudio = {version = "^2.2.1", source="torch"}
 torchvision = {version = "^0.17.1", source="torch"}
 ultralytics = {version = "^8.2.76", source = "PyPI"}
-wandb = {version = "^0.17.5", source = "PyPI"}
+wandb = {version = ">0.17.5", source = "PyPI"}
```

Comment:
Why > instead of ^?

Reply:
^ updates automatically but only minor releases, > updates also through major ones.
In this case there was an incompatibility so I had to change it.

Reply:
Ah it's because of the leading 0. I've always found it confusing why they treat ^0.x.y differently from ^1.x.y :D